pyproject.toml

```diff
@@ -83,10 +85,49 @@ packages = ["pastastore"]
 [tool.setuptools.dynamic]
 version = { attr = "pastastore.version.__version__" }
 
-[tool.black]
+[tool.ruff]
 line-length = 88
+extend-include = ["*.ipynb"]
 
-[tool.isort]
-profile = "black"
-src_paths = ["pastastore"]
-line_length = 88
+[tool.ruff.lint]
+# See: https://docs.astral.sh/ruff/rules/
+select = [
+    "C4",  # flake8-comprehensions
+    "E",   # pycodestyle
+    "F",   # pyflakes
+    "I",   # isort
+    "PT",  # pytest-style
+    "D",   # pydocstyle
+    "B",   # flake8-bugbear
+    "NPY", # numpy
+]
+
+[tool.ruff.format]
+
+[tool.ruff.lint.pydocstyle]
+convention = "numpy"
+
+[tool.pytest.ini_options]
+addopts = "--durations=0 --cov-report xml:coverage.xml --cov pastastore -v"
+markers = ["bgmap: test tile download for background map"]
+
+[tool.mypy]
```

Comment:
do we test type hinting with mypy?

Reply:
Not really I don't think, but you moved mypy ini into pyproject.toml, so we did have a mypy config file lying around, maybe for future use?